cli/src/pcluster/config/mappings.py

```diff
@@ -575,7 +577,19 @@
                 "default": "NONE",
                 "allowed_values": ALLOWED_VALUES["fsx_drive_cache_type"],
                 "update_policy": UpdatePolicy.UNSUPPORTED
-            })
+            }),
+            ("existing_mount_name", {
+                "type": FSxMountNameParam,
+                "default": "NONE",
+                "update_policy": UpdatePolicy.UNSUPPORTED,
```

Comment:
probably for private parameters we want to have IGNORED as UpdatePolicy and rely on the UNSUPPORTED update for fsx_fs_id?otherwise I don't know if this causes problems such as reporting them in the update diffs